### DISCUSSION TITLE
Can you add lyrics to it?

### DISCUSSION THREAD

**Discussion by IAMSHIVATRIPATHI** [namidaco/namida#228] [General]
I mean lyrics are visible side by side

**Comment by MSOB7YY** [comment]
i dont get it. any screenshots or explanation pls

**Reply by mmdoff1** [reply]
I guess he wants you to add full song lyrics when streaming?

**Reply by MSOB7YY** [reply]
@mmdoff1 lyrics when playing youtube can be obtained when u disable youtube-style miniplayer. i dont think thats what he asked
